### DISCUSSION TITLE
German Translation not working

### DISCUSSION THREAD

**Discussion by eliasu** [statamic/cms#4381] [Troubleshooting] [answered]
Hi there, 

short and simple: I changed `config/statamic/cp.php:83` to `"locale' => 'de"`
Sadly, it doesn't work as I supposed (see screenshot). Shouldn't the labels in the sidbar be translated as well? I think if seen that somewhere …

<img width="629" alt="Bildschirmfoto 2021-10-04 um 20 43 34" src="https://user-images.githubusercontent.com/11287110/135906906-6dfd8aff-6c3f-4dd9-806a-bc4b1914fe8e.png">

Is that supposed to be like that, or can you provide help?
Thank you!

**Comment by jasonvarga** [comment]
Can you post your `config/app.php` and `config/statamic/cp.php` files?

**Reply by eliasu** [reply]
Sure. Here you go …

[Archiv.zip](https://github.com/statamic/cms/files/7280964/Archiv.zip)

**Reply by jasonvarga** [reply]
So is the only word in English "Collections"? What else? For me it's working as expected.

![image](https://user-images.githubusercontent.com/105211/135920010-ce959f28-f5ef-4c70-9328-7a81f0e5424a.png)

Do you have any custom translation files in `resources/lang`?

**Reply by eliasu** [reply]
Actually, it's quite weird. Only some elements are translated, others not. Have a look:
All labels of the sidebar are not translated / some buttons are, some not / all explanations are translated …
I did not change anything in `resources/lang` … see my screenshot.

<img width="1029" alt="Bildschirmfoto 2021-10-04 um 22 33 48" src="https://user-images.githubusercontent.com/11287110/135920886-3c1000c1-af60-4ff6-8e3f-ed59f082031f.png">

<img width="955" alt="Bildschirmfoto 2021-10-04 um 22 30 03" src="https://user-images.githubusercontent.com/11287110/135920691-b201578b-86b6-4def-af45-7360eb4110a3.png">
<img width="1246" alt="Bildschirmfoto 2021-10-04 um 22 30 26" src="https://user-images.githubusercontent.com/11287110/135920696-6eb4c9ed-9f73-422e-b69e-ed3fbcba75b7.png">
<img width="189" alt="Bildschirmfoto 2021-10-04 um 22 29 55" src="https://user-images.githubusercontent.com/11287110/135920699-6621d29e-f6c5-4b86-8fa9-bd2bc1e135d1.png">

![Bildschirmfoto 2021-10-04 um 22 35 22](https://user-images.githubusercontent.com/11287110/135921372-e48b7b06-aec6-4151-9cc3-70fa1f21abd7.png)

**Reply by jasonvarga** [reply]
Do you have a `locale` in your user's yaml file?

**Reply by eliasu** [reply]
No … already checked it. Strange, huh?
Do you have multiple folders within `ressources/lang`?

**Comment by eliasu** [comment, marked as answer]
Ok, I got it. Thanks @jasonvarga for the help.
Maybe i misunderstood something, but I had to download the folder with the translation files here: [Statamics Github](https://github.com/statamic/cms/tree/3.2/resources/lang) and put that folder (with JSON file) into my projects `resources/lang` folder. 

Now it works as expected.

**Reply by jasonvarga** [reply]
You shouldn't need to do that. Weird.

**Comment by SirNovi** [comment]
So, I was in the exact same position. I think this might actually be a bug. 

No language would work properly, there was always only a small amount of elements that would get translated. I cleared all caches I could think of, to no avail.

Copying the language files from the vendor files directly to `resources/lang` fixes it, of course that's not an optimal solution in regards to updates.
